models/BaseEntity.cfc

```diff
@@ -749,7 +765,8 @@ component accessors="true" {
     }
 
     private function metadataInspection() {
-        var md = getMetadata( this );
+        var util = createObject("component","coldbox.system.core.util.Util");
```

Comment:
Can we inject this instead of creating it inline?

Reply:
I took my cues from how it's used in the framework. It's a one-off whenever it's used.  I can take a look at injection, though, since `BaseEntity` is a transient, it shouldn't be an issue.

Reply:
I think perhaps it can't be injected because WireBox hasn't injected anything yet — `init` vs. `onDIComplete`.

It really would be ideal if this were a separate package we could pull in as a dependency, but I'm not going to force that point.